Configuration/NodeTypes.yaml

```diff
@@ -0,0 +1,10 @@
+'TYPO3.Neos:Node':
+  properties:
+    restrictNode:
+      type: boolean
+      ui:
+        label: 'Flowpack.Neos.FrontendLogin:NodeTypes.Node:properties.restrictNode'
+        reloadIfChanged: TRUE
+        inspector:
+          group: 'visibility'
+          position: 100
```

Comment:
instead of adding this to all nodes, I'd suggest that you create a mixin and add the mixin as a super type for the general node type, since it's more clear and then you can remove it from a node type by setting that super type to false.. now you would unset it by setting ``properties.restrictNode`` to ``null`` / ``|``